dali/pipeline/data/types.h

```diff
@@ -118,6 +118,7 @@ enum DALIDataType : int {
   DALI_TENSOR_LAYOUT     = 23,
   DALI_PYTHON_OBJECT     = 24,
   DALI_TENSOR_LAYOUT_VEC = 25,
+  DALI_INTERLEAVE_TYPE   = 26,
```

Comment:
Recently we've been just using strings - this type's usability does not extend much beyond one specific operator.

Reply:
I will change it to a string then. Is there any requirement on those, i.e. upper / lower case or is there a parser that handles this?

Reply:
The strings are case sensitive and we use lowercase strings - see `Resize` and its `mode` argument values:
```C++
"default"
"stretch"
"not_larger"
"not_smaller"
```

Reply:
Done.